k8s/namespaces/ccd/ccd-case-document-am-api/aat.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
Need to add this patch to kustomize too:
https://github.com/hmcts/cnp-flux-config/blob/9996a0513e6ec0b73ddd360575ab89b9fe7122a4/k8s/aat/common-overlay/ccd/kustomization.yaml#L10